Description: Fixes spelling error ReEncode -> Re-encode
Author: Stefan Ritter <xeno@thehappy.de>
Bug: https://sourceforge.net/tracker/?func=detail&aid=3460901&group_id=194114&atid=948130
--- a/src/iso.c
+++ b/src/iso.c
@@ -22,7 +22,7 @@
 	printf("%%  iso.ps  reencodeing of ISO 8859-1 8-bit french characters\n");
 	printf("%%\n");
 	printf("/reencodedict 5 dict def\n");
-	printf("/ReEncode\n");
+	printf("/Re-encode\n");
 	printf("  { reencodedict begin\n");
 	printf("      /newencoding exch def\n");
 	printf("      /newfontname exch def\n");
@@ -246,5 +246,5 @@
 void re_encode (font)
 char *font;
 	{
-	printf("/%s /%s ISO ReEncode\n", font, font); 
+	printf("/%s /%s ISO Re-encode\n", font, font); 
 	}
